src/XMakeTasks/UnitTests/GenerateResourceOutOfProc_Tests.cs

```diff
@@ -1895,8 +1895,8 @@ public void SourceItemMissing()
         ///  Non-existent StateFile yields message
         /// </summary>
         [Fact]
-        [Trait("Category", "mono-osx-failing")]
         [Trait("Category", "mono-windows-failing")]
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Why make this windows specific? Previous traits suggests that it only affected mono on windows and osx.